testsettings.py

```diff
@@ -15,6 +15,7 @@
 NOSE_ARGS = [
     #'--no-migrations' # trim ~120s from test run with db tests
     #'--with-fixture-bundling',
+    '--with-doctest',
```

Comment:
I had [given up on this](https://github.com/dimagi/commcare-hq/pull/11568/commits/3d9e809f1bfc1d861f5532f86e31fd8b21ffc071) a while ago because [doctests are bad](http://nedbatchelder.com/blog/200811/things_i_dont_like_about_doctest.html) (and slow to find) and we don't have enough to make it worth trying to maintain it (@orangejenny and I had come to that conclusion after discussing).